docs/index.markdown

```diff
@@ -28,7 +28,7 @@ feature_row:
   - image_path: /static/feature_playbooks.png
     alt: "100% free"
     title: "Playbooks"
-    excerpt: "See all **26** sets of steps 🐾 to automatically response to a threat."
+    excerpt: "See all **31** sets of steps 🐾 to automatically response to a threat."
```

Comment:
Can we please replace this with:

See all **31** automated investigation and response playbooks.

Or something like that?

Reply:
Modified language here.